### ISSUE TITLE
NTR, pulmo: alveolar septal thickening

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6395]
Preferred term label:
(non-fibrotic?)  alveolar septal thickening

Synonyms
Alveolar septal thickening without fibrosis

Definition (free text, please give PubMed ID)
thickening of alveolar septa without significant cellular infiltrate

ISBN 978-1-4511-7643-8 (alk. paper)
Churg, Andrew, and Churg. Atlas of Interstitial Lung Disease Pathology : Pathology with High Resolution CT Correlations, Wolters Kluwer, 2013.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary alveolar system morphology HP:0033137
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

Your nano-attribution (ORCID)
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
This is related to this term -- https://hpo.jax.org/app/browse/term/HP:0030879
can we discuss how to represent this?

**Comment by pnrobinson** [comment]
This should be the (distinct) histo finding

**Comment by pnrobinson** [comment]
One question -- if the term is Alveolar septal thickening without fibrosis, then I do not understand why the definition ("without significant cellular infiltrate") is correct -=- if we do not have more cells or more scarring, then what is the correlate of the thickening?

**Comment by pnrobinson** [comment]
-single parent : 
Abnormal pulmonary alveolar system morphology HP:0033137


put https://hpo.jax.org/app/browse/term/HP:0025176 under HRCT hierarchy

**Comment by pnrobinson** [comment]
added
